director/lib/converter_facade.py

```diff
@@ -380,7 +381,9 @@ class ConverterContext(typing.NamedTuple):
 class ConverterFacade(object):
     converter_binary: typing.List[str]
 
-    def __init__(self, converter_binary: typing.List[str]) -> None:
+    def __init__(
+        self, converter_binary: typing.List[str] = settings.STENCILA_BINARY
```

Comment:
I personally don't like this pattern. The point of ConverterFacade is to decouple it from Django.

Reply:
OK, I've removed this.